### PR TITLE
Document all of the things

### PR DESCRIPTION
**Pull request by brianamarie**
Based on the great work done [on teacher-bot proper by @crichID](https://github.com/teacher-bot/teacherbot/pull/4/) and the continuing work in [teacher-bot/add-collabs](https://github.com/teacher-bot/add-collabs/pull/4), this pull request attempts to add documentation to match the [probot/plugin-template](https://github.com/probot/plugin-template). 

I will `@mention` once it's ready for review. 👍 

ToDo: Review all files to make sure they match 🤷‍♀️ 
- [x] docs/deploy
- [x] .env.example
- [x] .gitignore
- [x] .travis.yml
- [x] LICENSE 
- [x] README.md
- [x] app.json
- [x] index.js
- [x] package.json
- ~~`package-lock.json` = delete package-lock.json file~~ Unnecessary, it didn't exist
- [x] Remove .travis.yml file and app.json file
- [x] README with the appropriate information on configuration and the permission

### CODE REVIEW

**Comment by brianamarie** [comment]
@teacher-bot/training-team This is ready for review as well 🎉 

Two same questions as in [the other PR](https://github.com/teacher-bot/add-collabs/pull/4): 
- Do we want to run Travis on this repository?
- The package-lock.json file isn't updated with the new info in package.json. Is this a big deal?

**Comment by crichID** [comment]
@brianamarie I'm not seeing the README in the diff ... am I missing something?

**Comment by brianamarie** [comment]
_Removed comment here to update OP_

**Comment by brianamarie** [comment]
> @brianamarie I'm not seeing the README in the diff ... am I missing something?

@crichID I need to go through with a fine tooth comb (and @JasonEtco & @hectorsector's help) for both this README and the one in `add-collabs` repo to make sure the README.md is accurate.

**Comment by brianamarie** [comment]
@teacher-bot/training-team Since our meeting, I've
- deleted `app.json`
- deleted `.travis.yml` 
- added permissions (best guess) to `README.md` 

This is ready for review and 👀, especially the `README.md` permissions.

File: README.md [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
+### Configuring Customized Responses
 
-You can use the [default responses](lib/defaults.js), but if you'd like to specify your own, create a file inside a `.github` folder, titled `teacherbot.yml` and include the following text:
+You can use the [default responses](lib/defaults.js), but if you'd like to specify your own, create a file inside a `.github` folder, named `teacherbot.yml` and include the following text:
 
 ```yml
-addCollaborators:
-  newCollaboratorMessage: "This message will appear when someone opens a new issue, and is not already a collaborator."
-  existingCollaboratorMessage: "This message appears when someone is already a collaborator and they open a new issue."
+remindMerge: {
```
Comment (crichID) [inline README.md]: @brianamarie I noticed we didn't include the curly brackets in the other examples. Are they needed here?

File: README.md [hunk 2]
```diff
@@ -1,38 +1,39 @@
...
+    message: ':wave: hiya Please remember to delete your branch after merging or closing if you haven\'t done so already.'
```
Comment (crichID) [inline README.md]: @brianamarie we also used " instead of ' in the other examples, is it possible to use " here?

**Review by crichID — CHANGES_REQUESTED** [review]
@brianamarie I left a couple small comments just to see if we can make the formatting of the .yml files consistent -- otherwise, this looks great!

**Comment by brianamarie** [comment]
@crichID Great points - I have addressed them in 6fe8f24.

**Review by crichID — APPROVED** [review]

**Comment by crichID** [comment]
Looks beautiful @brianamarie thank you! :shipit:

**Pull request merged**